core/src/Backdrop.html

```diff
@@ -1,10 +1,9 @@
 {#if backdropActive}
-    <div
-     class="fd-ui__overlay fd-overlay fd-overlay--modal {heightCssClass}"
-     aria-hidden="false"
-    >
-    </div>
+<div class="fd-ui__overlay fd-overlay fd-overlay--modal {heightCssClass}" aria-hidden="false">
```

Comment:
Why do we need these slot tags here?

Reply:
Because we wrap App's content in `<Backdrop>` tags. `<slot>` is for displaying what's inside component tags. It's the same as `<ng-content>` in Angular. Am I wrong?